### PR TITLE
python3Packages.tensorboard: 2.20.0 -> 2.21.0

### PR DESCRIPTION
**Pull request by GaetanLepage**
## Things done

Diff: https://github.com/tensorflow/tensorboard/compare/2.20.0...2.21.0
Changelog: https://github.com/tensorflow/tensorboard/releases/tag/2.21.0

- Built on platform:
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [x] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [ ] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.
- [ ] Follows the [automation/AI policy].

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[automation/AI policy]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#automationai-policy
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

### CODE REVIEW

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 536795`
Commit: `95569f23bd94c97083ce3c60e6083c6d0a8660ff`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 88 packages built:</summary>
  <ul>
    <li>calibre</li>
    <li>convertx</li>
    <li>frigate</li>
    <li>photoprism</li>
    <li>piper-tts</li>
    <li>python313Packages.ale-py</li>
    <li>python313Packages.array-record</li>
    <li>python313Packages.ax-platform</li>
    <li>python313Packages.bambi</li>
    <li>python313Packages.baselines</li>
    <li>python313Packages.botorch</li>
    <li>python313Packages.brax</li>
    <li>python313Packages.cleanlab</li>
    <li>python313Packages.clu</li>
    <li>python313Packages.deepface</li>
    <li>python313Packages.distrax</li>
    <li>python313Packages.dm-control</li>
    <li>python313Packages.dm-sonnet</li>
    <li>python313Packages.edward</li>
    <li>python313Packages.energyflow</li>
    <li>python313Packages.etils</li>
    <li>python313Packages.evosax</li>
    <li>python313Packages.experiment-utilities</li>
    <li>python313Packages.flax</li>
    <li>python313Packages.flowjax</li>
    <li>python313Packages.gpt-2-simple</li>
    <li>python313Packages.gymnasium</li>
    <li>python313Packages.imgcat</li>
    <li>python313Packages.keras</li>
    <li>python313Packages.kserve</li>
    <li>python313Packages.lerobot</li>
    <li>python313Packages.minari</li>
    <li>python313Packages.mlcroissant</li>
    <li>python313Packages.mtcnn</li>
    <li>python313Packages.mujoco</li>
    <li>python313Packages.mujoco-mjx</li>
    <li>python313Packages.numpyro</li>
    <li>python313Packages.nutpie</li>
    <li>python313Packages.optuna-dashboard</li>
    <li>python313Packages.orbax-checkpoint</li>
    <li>python313Packages.orbax-export</li>
    <li>python313Packages.outlines</li>
    <li>python313Packages.pettingzoo</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pymanopt</li>
    <li>python313Packages.pytorch-metric-learning</li>
    <li>python313Packages.retina-face</li>
    <li>python313Packages.rlax</li>
    <li>python313Packages.shimmy</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.snorkel</li>
    <li>python313Packages.stable-baselines3</li>
    <li>python313Packages.syne-tune</li>
    <li>python313Packages.tensorboard</li>
    <li>python313Packages.tensorboardx</li>
    <li>python313Packages.tensorflow (python313Packages.tensorflow-bin, python313Packages.tensorflowWithoutCuda)</li>
    <li>python313Packages.tensorflow-datasets</li>
    <li>python313Packages.tensorflow-probability</li>
    <li>python313Packages.tensorflowWithCuda</li>
    <li>python313Packages.tf-keras</li>
    <li>python313Packages.tf2onnx</li>
    <li>python313Packages.tflearn</li>
    <li>python313Packages.tianshou</li>
    <li>python313Packages.torch-tb-profiler</li>
    <li>python313Packages.torchrl</li>
    <li>python313Packages.torchtitan</li>
    <li>python313Packages.torchtnt</li>
    <li>python313Packages.trainer</li>
    <li>python313Packages.vllm</li>
    <li>python313Packages.vmas</li>
    <li>python313Packages.waymax</li>
    <li>python313Packages.whisperx</li>
    <li>python314Packages.experiment-utilities</li>
    <li>python314Packages.pyannote-audio</li>
    <li>python314Packages.pytorch-metric-learning</li>
    <li>python314Packages.snorkel</li>
    <li>python314Packages.tensorboard</li>
    <li>python314Packages.tensorboardx</li>
    <li>python314Packages.torch-tb-profiler</li>
    <li>python314Packages.torchtitan</li>
    <li>python314Packages.torchtnt</li>
    <li>python314Packages.trainer</li>
    <li>python314Packages.whisperx</li>
    <li>unbook</li>
    <li>vllm</li>
    <li>whisper-ctranslate2</li>
    <li>whisperx</li>
    <li>wyoming-piper</li>
  </ul>
</details>


---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 79 packages built:</summary>
  <ul>
    <li>calibre</li>
    <li>convertx</li>
    <li>frigate</li>
    <li>photoprism</li>
    <li>piper-tts</li>
    <li>python313Packages.ale-py</li>
    <li>python313Packages.ax-platform</li>
    <li>python313Packages.bambi</li>
    <li>python313Packages.baselines</li>
    <li>python313Packages.botorch</li>
    <li>python313Packages.brax</li>
    <li>python313Packages.cleanlab</li>
    <li>python313Packages.deepface</li>
    <li>python313Packages.distrax</li>
    <li>python313Packages.dm-control</li>
    <li>python313Packages.edward</li>
    <li>python313Packages.energyflow</li>
    <li>python313Packages.etils</li>
    <li>python313Packages.evosax</li>
    <li>python313Packages.experiment-utilities</li>
    <li>python313Packages.flax</li>
    <li>python313Packages.flowjax</li>
    <li>python313Packages.gpt-2-simple</li>
    <li>python313Packages.gymnasium</li>
    <li>python313Packages.imgcat</li>
    <li>python313Packages.keras</li>
    <li>python313Packages.lerobot</li>
    <li>python313Packages.minari</li>
    <li>python313Packages.mlcroissant</li>
    <li>python313Packages.mtcnn</li>
    <li>python313Packages.mujoco</li>
    <li>python313Packages.mujoco-mjx</li>
    <li>python313Packages.numpyro</li>
    <li>python313Packages.nutpie</li>
    <li>python313Packages.optuna-dashboard</li>
    <li>python313Packages.orbax-checkpoint</li>
    <li>python313Packages.orbax-export</li>
    <li>python313Packages.outlines</li>
    <li>python313Packages.pettingzoo</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pymanopt</li>
    <li>python313Packages.pytorch-metric-learning</li>
    <li>python313Packages.retina-face</li>
    <li>python313Packages.rlax</li>
    <li>python313Packages.shimmy</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.snorkel</li>
    <li>python313Packages.stable-baselines3</li>
    <li>python313Packages.syne-tune</li>
    <li>python313Packages.tensorboard</li>
    <li>python313Packages.tensorboardx</li>
    <li>python313Packages.tensorflow (python313Packages.tensorflow-bin, python313Packages.tensorflowWithoutCuda)</li>
    <li>python313Packages.tensorflow-probability</li>
    <li>python313Packages.tf-keras</li>
    <li>python313Packages.tf2onnx</li>
    <li>python313Packages.tflearn</li>
    <li>python313Packages.tianshou</li>
    <li>python313Packages.torch-tb-profiler</li>
    <li>python313Packages.torchtitan</li>
    <li>python313Packages.torchtnt</li>
    <li>python313Packages.trainer</li>
    <li>python313Packages.vmas</li>
    <li>python313Packages.waymax</li>
    <li>python313Packages.whisperx</li>
    <li>python314Packages.experiment-utilities</li>
    <li>python314Packages.pyannote-audio</li>
    <li>python314Packages.pytorch-metric-learning</li>
    <li>python314Packages.snorkel</li>
    <li>python314Packages.tensorboard</li>
    <li>python314Packages.tensorboardx</li>
    <li>python314Packages.torch-tb-profiler</li>
    <li>python314Packages.torchtitan</li>
    <li>python314Packages.torchtnt</li>
    <li>python314Packages.trainer</li>
    <li>python314Packages.whisperx</li>
    <li>unbook</li>
    <li>whisper-ctranslate2</li>
    <li>whisperx</li>
    <li>wyoming-piper</li>
  </ul>
</details>


---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 41 packages built:</summary>
  <ul>
    <li>piper-tts</li>
    <li>python313Packages.baselines</li>
    <li>python313Packages.energyflow</li>
    <li>python313Packages.etils</li>
    <li>python313Packages.experiment-utilities</li>
    <li>python313Packages.gpt-2-simple</li>
    <li>python313Packages.imgcat</li>
    <li>python313Packages.mlcroissant</li>
    <li>python313Packages.orbax-checkpoint</li>
    <li>python313Packages.orbax-export</li>
    <li>python313Packages.outlines</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pymanopt</li>
    <li>python313Packages.pytorch-metric-learning</li>
    <li>python313Packages.snorkel</li>
    <li>python313Packages.stable-baselines3</li>
    <li>python313Packages.tensorboard</li>
    <li>python313Packages.tensorboardx</li>
    <li>python313Packages.tensorflow (python313Packages.tensorflow-bin, python313Packages.tensorflowWithoutCuda)</li>
    <li>python313Packages.tf-keras</li>
    <li>python313Packages.tflearn</li>
    <li>python313Packages.tianshou</li>
    <li>python313Packages.torch-tb-profiler</li>
    <li>python313Packages.torchtnt</li>
    <li>python313Packages.trainer</li>
    <li>python313Packages.whisperx</li>
    <li>python314Packages.experiment-utilities</li>
    <li>python314Packages.pyannote-audio</li>
    <li>python314Packages.pytorch-metric-learning</li>
    <li>python314Packages.snorkel</li>
    <li>python314Packages.stable-baselines3</li>
    <li>python314Packages.tensorboard</li>
    <li>python314Packages.tensorboardx</li>
    <li>python314Packages.tianshou</li>
    <li>python314Packages.torch-tb-profiler</li>
    <li>python314Packages.torchtnt</li>
    <li>python314Packages.trainer</li>
    <li>python314Packages.whisperx</li>
    <li>whisper-ctranslate2</li>
    <li>whisperx</li>
    <li>wyoming-piper</li>
  </ul>
</details>

**Review by wegank — APPROVED** [review]
